src/utils.js

```diff
@@ -24,7 +28,17 @@ const getAccessToken = () => {
   return cookies.get('access_token');
 };
 
+// Configure axios with response interceptor
+axios.interceptors.response.use(response => response, (error) => {
+  const errorStatus = error.response.status;
+  if (errorStatus === 401 || errorStatus === 403) {
+    store.dispatch(logout());
+  }
+  return Promise.reject(error);
+});
+
 export {
+  axios,
```

Comment:
I think we should call this something different, maybe `httpClient` so we know its not stock axios

Reply:
Good idea!